tests/unit/test_rat_board.py

```diff
@@ -132,3 +135,12 @@ async def test_modify_rescue_explosion(rat_board_fx, random_string_fx):
             raise RuntimeError  # intentionally raise an exception
 
     assert random_string_fx in rat_board_fx, "the board dropped the rescue!"
+
+@pytest.mark.asyncio
+async def test_modify_rescue_datetimelastcase(rat_board_fx, monkeypatch):
```

Comment:
[style] snake_case between words in a function definition please.
```suggestion
async def test_modify_rescue_datetime_last_case(rat_board_fx, monkeypatch):
```